openapi/atlas-api.yaml

```diff
@@ -17387,7 +17387,7 @@ paths:
       description: Return all non-compliant resources for an org.
       externalDocs:
         description: Atlas Resource Policies
-        url: https://docs.atlas.mongodb.com
+        url: https://dochub.mongodb.org/core/atlas-resource-policies
```

Comment:
these dochub URLs don't seem correct, they don't work for me

Reply:
Pinged relevant team.